subprojects/docs/src/main/resources/header.html

```diff
@@ -95,92 +95,78 @@
-            <li><a class="nav-dropdown" data-toggle="collapse" href="#organizing-gradle-projects" aria-expanded="false" aria-controls="organizing-gradle-projects">Organizing Project Structure</a>
-                <ul id="organizing-gradle-projects">
-                    <li><a href="/userguide/organizing_gradle_projects.html">Organizing Projects Best Practices</a></li>
-                    <li><a href="/userguide/directory_layout.html">Directory Layout</a></li>
-                    <li><a href="/userguide/composite_builds.html">Integrating Separate Gradle Builds (Composite Builds)</a></li>
-                    <li><a href="/userguide/intro_multi_project_builds.html">Multi-Project Builds</a></li>
+            <li>Using the Command Line (TBD)</li>
+            <li>Using Gradle From an IDE (TBD)</li>
+            <li><a class="nav-dropdown" data-toggle="collapse" href="#customizing-execution" aria-expanded="false" aria-controls="customizing-execution">Customizing Execution</a>
+                <ul id="customizing-execution">
+                    <li><a href="/userguide/build_environment.html">Configuring the Build Environment</a></li>
+                    <li><a href="/userguide/gradle_daemon.html">Configuring the Gradle Daemon</a></li>
+                    <li><a href="/userguide/init_scripts.html">Creating Initialization Scripts</a></li>
                 </ul>
             </li>
+            <li><a href="/userguide/intro_multi_project_builds.html">Executing Multi-Project Builds</a></li>
+            <li><a href="https://guides.gradle.org/creating-build-scans/">Inspecting Gradle Builds</a></li>
             <li><a class="nav-dropdown" data-toggle="collapse" href="#optimizing-build-performance" aria-expanded="false" aria-controls="optimizing-build-performance">Optimizing Build Times</a>
                 <ul id="optimizing-build-performance">
                     <li><a href="https://guides.gradle.org/performance/">Build Performance Guide</a></li>
                     <li><a href="/userguide/build_cache.html">Enabling and Configuring the Build Cache</a></li>
-                    <li><a href="https://guides.gradle.org/using-build-cache/">Using the Build Cache</a></li>
                 </ul>
             </li>
-            <li><a class="nav-dropdown" data-toggle="collapse" href="#customizing-execution" aria-expanded="false" aria-controls="customizing-execution">Customizing Execution</a>
-                <ul id="customizing-execution">
-                    <li><a href="/userguide/build_environment.html">Configuring the Build Environment</a></li>
-                    <li><a href="/userguide/gradle_daemon.html">Configuring the Gradle Daemon</a></li>
-                    <li><a href="/userguide/init_scripts.html">Initialization Scripts</a></li>
-                </ul>
-            </li>
-
-            <li><a href="/userguide/gradle_wrapper.html">Provisioning Gradle (Wrapper)</a></li>
```

Comment:
I don't understand what "Provisioning" means in this instance, and I know Gradle fairly well. I think the terminology would be confusing for new and average users.

I've always thought that the Gradle Wrapper should simply be introduced through other chapters and content. And this chapter goes into more depth than is relevant to most users. That makes it a great candidate for the appendix, which is where I moved it to.